libgraph/include/katana/GraphTopology.h

```diff
@@ -311,6 +318,26 @@ class KATANA_EXPORT EdgeShuffleTopology : public GraphTopology {
     return ret;
   }
 
+  static std::shared_ptr<EdgeShuffleTopology> Make(
+      PropertyGraph* pg, const RDGTopology::TransposeKind& tpose_todo,
+      const RDGTopology::EdgeSortKind& edge_sort_todo,
```

Comment:
Same here. enum isn't needed for now. You can rename the method to `MakeEdgesSortedByPropety`